src/Layout.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+import templates from './Layout.soy.js';
+import Component from 'metal-component';
+import Soy from 'metal-soy';
+
+class Layout extends Component {
+}
+
+Layout.STATE = {
+ data: []
```

Comment:
This is not in the right format, it probably only worked because we automatically add the params in the soy template as state properties for you. It's still good to list them there so they can be documented though.

The right format would be something like this:

```
import 'core' from 'metal';

//...

Layout.STATE = {
  /**
   * Add docs explaining what `data` is. Take a look at https://github.com/metal/metal-pagination/blob/master/src/Pagination.js#L169 as an example.
   * @type {!Array}
   */
  data: {
    validator: core.isArray,
    valueFn: () => []
  }
};
```